### PR TITLE
Remove verificationMethod from VerifyOptions

### PR DESCRIPTION
**Pull request by peacekeeper**
When verifying, being able to specify a verificationMethod as an option can be dangerous, see discussion in https://github.com/w3c-ccg/vc-api/pull/248.

### CODE REVIEW

**Review by clehner — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

File: components/VerifyOptions.yml [hunk 1]
```diff
@@ -17,12 +17,6 @@ components:
       additionalProperties: false
       description: Options for specifying how the LinkedDataProof is verified.
       properties:
-        verificationMethod:
-          type: string
-          description: The URI of the verificationMethod used for the proof. If omitted, a default verification method will be used.
-        proofPurpose:
-          type: string
-          description: The purpose of the proof. Default 'assertionMethod'.
```
Comment (msporny) [inline components/VerifyOptions.yml:25]: I'm suggesting we remove this as well. You shouldn't be able to override `verificationMethod` or `proofPurpose` when verifying.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
@peacekeeper -- I'm expanding the scope of this PR to also remove overriding `proofPurpose` -- are you ok with that? Feel free to put it back, but if you do, I'll just end up raising another PR to remove it.

We need @dlongley @OR13 input on this one.

**Review by dlongley — APPROVED** [review]

**Comment by peacekeeper** [comment]
> are you ok with that?

Yes I agree this makes sense.

**Comment by peacekeeper** [comment]
Merged this after multiple approvals and no objections.

**Pull request merged**